### ISSUE TITLE
Ammo tracking on weapon display

### ISSUE THREAD

**Issue by SerotoninRonin** [fvtt-fria-ligan/twilight2000-foundry-vtt#138]
### Description

When playing Twilight:2000 on FoundryVTT I would like to be able to track the current loaded ammunition amount on the weapon.  Currently there exists a weapon ammunition column in the weapon display "MAG" which shows how much a Magazine/Reload should hold and a separate item for each Mag/Reload at the bottom of the screen.  

### Suggested solution

Change the MAG column to display X/Y where X is the current number of rounds in the currently loaded Reload/Mag and Y is the potential full amount the Mag/Reload can hold. 
Additionally a button/icon on the left that allows you to activate a reload from the item without having to open and edit the item sheet and assign a new reload would be a fantastic QOL addition. 

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by tinwe** [comment]
Really awesome quality of life suggestions!

**Comment by andaug** [comment]
I agree, really missing this. Super annoying to have to open the weapon description to see it, or having to install modules that always run out of support.
